excalidraw-app/App.tsx

```diff
@@ -364,6 +365,21 @@ const ExcalidrawWrapper = () => {
   });
   const collabError = useAtomValue(collabErrorIndicatorAtom);
 
+  const isLocalStorageOverflow = useAtomValue(isLocalStorageOverflowAtom);
+
+  useEffect(() => {
+    if (isLocalStorageOverflow) {
+      if (excalidrawAPI) {
+        excalidrawAPI.setToast({
```

Comment:
You could just skip the `if` and do `excalidrawAPI?.setToast`

Reply:
Fixed!